src/SharpBrick.PoweredUp/Hubs/MoveHub.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using Microsoft.Extensions.Logging;
+using SharpBrick.PoweredUp.Devices;
+using SharpBrick.PoweredUp.Protocol;
+
+namespace SharpBrick.PoweredUp
+{
+    public class MoveHub : Hub
+    {
+        public MoveHub(ILegoWirelessProtocol protocol, IDeviceFactory deviceFactory, ILogger<MoveHub> logger, IServiceProvider serviceProvider = default)
+            : base(protocol, deviceFactory, logger, serviceProvider, SystemType.LegoSystem_MoveHub, new Port[] {
+                new Port(0, nameof(A), false, expectedDevice: DeviceType.InternalMotorWithTacho),
+                new Port(1, nameof(B), false, expectedDevice: DeviceType.InternalMotorWithTacho),
+                // Since ports C and D can be any compatible sensor or motor, we don't set an expected device type here
+                new Port(2, nameof(C), true),
+                new Port(3, nameof(D), true),
+                new Port(16, nameof(AB), false, expectedDevice: DeviceType.InternalMotorWithTacho, true),
+                new Port(50, string.Empty, false, expectedDevice: DeviceType.RgbLight),
+                new Port(58, string.Empty, false, expectedDevice: DeviceType.InternalTilt),
+                new Port(59, string.Empty, false, expectedDevice: DeviceType.Current),
+                new Port(60, string.Empty, false, expectedDevice: DeviceType.Voltage),
+                new Port(70, string.Empty, false)
```

Comment:
We can take it out or leave a comment but will need a null check in OnHubAttachedIOMessage handling since the hub will notify and there won't be a port to get.  If in that case it is fine to just ignore any ports that aren't exposed in the hub then I can make that change easily enough.

Reply:
Thought about it some more and removed it for now and added null check into OnHubAttachedIOMessage method in Hub so does not generate exception.